cmd/krel/cmd/ff.go

```diff
@@ -75,36 +70,31 @@ func init() {
 func runFf(opts *ffOptions) error {
 	// TODO: Add usage/help
 	// TODO: Set positional args
-	// TODO: Fail on empty branch
 	// TODO: Fail on GITHUB_TOKEN not set
 
 	if !command.Available("git", "go", "make", "jq") {
 		log.Fatalf("Unable to meet executable dependency requirements")
 	}
 
 	branch := opts.branch
+	if branch == "" {
+		return errors.New("Please specify valid release branch")
+	}
 	masterRef := opts.masterRef
-	remote := kgit.DefaultRemote
-	remoteMaster := fmt.Sprintf("%s/%s", remote, "master")
+	remoteMaster := kgit.Remotify("master")
 
 	log.Printf("Preparing to fast-forward master@%s onto the %s branch...\n", masterRef, branch)
 
-	nomock := rootOpts.nomock
-	dryRunFlag := "--dry-run"
-	if nomock {
-		// TODO: Set this to empty string once we're ready to turn on the tool
-		log.Println("Running in no-mock mode!")
-		dryRunFlag = "--dry-run"
-	}
-
-	// TODO: Refactor everything to use this repo instance
 	repo, err := kgit.CloneOrOpenDefaultGitHubRepoSSH(rootOpts.repoPath)
 	if err != nil {
 		return err
 	}
 
-	isReleaseBranch := kgit.IsReleaseBranch(branch)
-	if !isReleaseBranch {
+	if !rootOpts.nomock {
+		repo.SetDry()
```

Comment:
We should signal here whether the tool is running in mock or no-mock mode.

Reply:
Fixed :+1: